lib/matplotlib/axes/_axes.py

```diff
@@ -5170,9 +5170,9 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
         shape : scalars (columns, rows), optional, default: None
             For raw buffer images
 
-        filternorm : scalar, optional, default: 1
-            A parameter for the antigrain image resize filter.  From the
-            antigrain documentation, if `filternorm` = 1, the filter
+        filternorm : bool, optional, default: True
```

Comment:
OK, so `filternorm` was either 1 or zero? 

The rest of this docstring doesn't make a lick of sense to me.  I think they are trying to say the sum of the filter should be one, but???

Reply:
Feel free to propose a better wording :-) (it doesn't make sense to me either)
At least since #10383 `matplotlib._image.resample` also documents `norm` as a bool.